os/arch/arm/src/armv7-a/Make.defs

```diff
@@ -0,0 +1,96 @@
+############################################################################
+# arch/arm/src/armv7-a/Make.defs
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.  The
+# ASF licenses this file to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance with the
+# License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+############################################################################
+
+# Common ARM files
+
+### Other architectures do not have their own Make.defs file
+#include common/Make.defs
+
+# The vector table is the "head" object, i.e., the one that must forced into
+# the link in order to draw in all of the other components
+
+HEAD_ASRC += arm_vectortab.S
+
+ifeq ($(CONFIG_BUILD_KERNEL),y)
```

Comment:
looks like this config is used at multiple places. Could you change it as mentioned above?